flocker/provision/_install.py

```diff
@@ -40,9 +40,118 @@
                 "centos/clusterhq-release$(rpm -E %dist).noarch.rpm".format(
                     archive_bucket=ARCHIVE_BUCKET,
                     ),
+    # FLOC-1828 TODO - use ARCHIVE_BUCKET rather than clusterhq-archive-testing
+    # This needs a release to be done to add .deb files to clusterhq-archive
+    'ubuntu-14.04': 'https://clusterhq-archive-testing.s3.amazonaws.com/'
+                'ubuntu/14.04/$(ARCH)'.format(archive_bucket=ARCHIVE_BUCKET),
```

Comment:
I'm not sure we need a comment about `$(ARCH)`, since it doesn't get expanded. (`apt` will expand it when it reads the generated file).